### ISSUE TITLE
Table View Redesigned

### ISSUE THREAD

**Issue by nitin10s** [aboutcode-org/scancode-workbench#451]
### Description

Redesigned the Table View
![Table View – 1](https://user-images.githubusercontent.com/16988558/81693473-93d11f00-947d-11ea-913b-c232c20e4907.png)

Made one more design with respect to the discussions on Gitter [Toggle for Activating Filters]
![Table View](https://user-images.githubusercontent.com/16988558/81693569-be22dc80-947d-11ea-8bde-ed98972d2355.png)

**Comment by steven-esser** [comment]
@nitin10s what does it look like with the rows in place?

**Comment by nitin10s** [comment]
![Table View](https://user-images.githubusercontent.com/16988558/82062264-7c956a00-96e7-11ea-9e4d-b85f95d84b23.png)
Something like this... a scrollbar on the bottom of the table as it is now.

**Comment by steven-esser** [comment]
This looks great 👍

**Comment by steven-esser** [comment]
@nitin10s Ping, I would be happy to review and merge any changes you have here. The visual design is quite nice!

The same goes for the other issues you have opened.

**Comment by mjherzog** [comment]
Wondering if combining the filter with column headings will impact the number of columns you can display.

**Comment by nitin10s** [comment]
@majurg Actually, I am just a UX designer and was wanting to suggest some redesigning with better user experience. Incase anyone volunteers to design these, I would be very happy to help him along the process providing all the necessary design specs required. 

@mjherzog Yes, it reduces of columns displayed at an instance. We can maybe restrict the filter box width with respect to the original column width.

Also, I would be very happy to take up any other projects that may require designing UX.

**Comment by steven-esser** [comment]
@nitin10s Ah that makes sense. Are all the assets created by yourself?

**Comment by nitin10s** [comment]
@majurg Yes, everything has been designed from scratch. Also, I am well versed with the frontend development tech and would be happy to collaborate and assist someone who takes up the implementation work 😃.

**Comment by chinmaym07** [comment]
Hey @majurg may I work on this redesigning work.?

**Comment by steven-esser** [comment]
@chinmaym07 Yes of course :) 

You will probably want to sync with @nitin10s, as he has been kind enough to offer his help to potential contributors.

**Comment by OmkarPh** [comment]
hi @pombredanne 
is this issue still relevant

**Comment by pombredanne** [comment]
@OmkarPh I think this is still relevant alright.
@nitin10s gentle ping

**Comment by nitin10s** [comment]
Hey, seems like I lost the file I had these designs in 😢 , I'll replicate the designs and share a figma file in sometime.

**Comment by pombredanne** [comment]
@nitin10s no worries, we have the mockups here as images still?
Or is a figma file something that has the CSS too?

**Comment by OmkarPh** [comment]
yes, there is css in figma files,
for example, I created this for recent file uploads - https://www.figma.com/file/Ifn6b54LIb135DORW8NgcR/Workbench-designs?node-id=0%3A1
you have to signup in order to view the css
